LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2015 President and Fellows of Harvard College
+Copyright (C) 2016 Python LTI Initiative
```

Comment:
I think we can/should just drop the copyright statement. For an analog org/project see https://github.com/pytest-dev/pytest/blob/master/LICENSE.